docs/reference/asyncapi-rules.md

```diff
@@ -219,6 +219,40 @@ components:
 
 **Recommended:** Yes
 
+### asyncapi-message-messageId-uniqueness
+
+`messageId` must be unique across all the messages (except these one defined in the components).
```

Comment:
Why not all messages? 🤔

Reply:
I've been thinking about it and the components can also be used in other specs and additionally some message do not have to be used at all inside the spec in which they are located, so I check only those that are "actively" involved in the spec, in described operations.

Reply:
That makes sense yea 🤔 

As long as those messages that are being referenced from the active operations are checked, but they are so all good 👍